packages/now-cli/src/util/dev/server.ts

```diff
@@ -780,16 +782,18 @@ export default class DevServer {
 
     // Updating builders happens lazily, and any builders that were updated
     // get their "build matches" invalidated so that the new version is used.
-    this.updateBuildersPromise = updateBuilders(builders, this.output)
-      .then(updatedBuilders => {
-        this.updateBuildersPromise = null;
-        this.invalidateBuildMatches(nowConfig, updatedBuilders);
-      })
-      .catch(err => {
-        this.updateBuildersPromise = null;
-        this.output.error(`Failed to update builders: ${err.message}`);
-        this.output.debug(err.stack);
-      });
+    this.updateBuildersTimeout = setTimeout(() => {
```

Comment:
Can this be called multiple times? If so, would it make sense to clear an existing timeout before we set another one?

Reply:
I don't believe so, this is in the `devserver.start()` function so it's only called once.